src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapper.java

```diff
@@ -25,9 +27,15 @@ public Bundle mapToBundle(final PathologyRecord pathologyRecord) {
         Bundle bundle = generateInitialPathologyBundle();
 
         bundle.addEntry()
-            .setFullUrl(FULL_URL_PREFIX.concat(uuidGenerator.generateUUID()))
+            .setFullUrl(FULL_URL_PREFIX.concat(pathologyRecord.getRequester().getId()))
             .setResource(pathologyRecord.getRequester());
 
+        Optional.ofNullable(pathologyRecord.getPerformer()).ifPresent(performer -> {
+            bundle.addEntry()
+                .setFullUrl(FULL_URL_PREFIX.concat(performer.getId()))
+                .setResource(performer);
+            }
+        );
```

Comment:
FYI: because the body of this lambda is a single expression, the braces can be removed
```suggestion
        Optional.ofNullable(pathologyRecord.getPerformer()).ifPresent(performer ->
            bundle.addEntry()
                .setFullUrl(FULL_URL_PREFIX.concat(performer.getId()))
                .setResource(performer));
```

Reply:
Removed this